templates/flat-template.svg

```diff
@@ -3,10 +3,17 @@
     <stop offset="0" stop-color="#bbb" stop-opacity=".1"/>
     <stop offset="1" stop-opacity=".1"/>
   </linearGradient>
-  <rect rx="3" width="{{=it.widths[0]+it.widths[1]}}" height="20" fill="{{=it.escapeXml(it.colorA||"#555")}}"/>
```

Comment:
For future reference, this is the left part of the badge, but it goes all the way to the right to match the border-radius. Ugly hack.